scripts/proto_splitter/proto2multi.py

```diff
@@ -0,0 +1,137 @@
+
```

Comment:
```suggestion
#!/usr/bin/env python

# Copyright 1996-2020 Cyberbotics Ltd.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

```

We need to add a license header to this file.
@Simon-Steinmann if you agree lets put Apache 2.

Reply:
Sure, but I would like to be mentioned as well.  I don't care much about the copyright. Perhaps we can add:
`Author 2020 Simon Steinmann (https://github.com/Simon-Steinmann)`

Reply:
Sure, what about this edited version?